modules/nw-osp-configuring-external-load-balancer.adoc

```diff
@@ -260,7 +254,30 @@ A record pointing to Load Balancer Front End
 DNS propagation might take some time for each DNS record to become available. Ensure that each DNS record propagates before validating each record.
 ====
 
-. Use the `curl` CLI command to verify that the external load balancer and DNS record configuration are operational:
+ifdef::bare-metal[]
+. For your {product-title} cluster to use the user-managed load balancer, you must specify the following configuration in your cluster's `install-config.yaml` file:
++
+[source,yaml]
+----
+# ...
+platform:
+  baremetal:
+    loadBalancer:
+      type: UserManaged <1>
+      apiVIPs:
+      - <api_ip> <2>
+      ingressVIPs:
+      - <ingress_ip> <3>
+# ...
+----
+<1> Set `UserManaged` for the `type` parameter to specify a user-managed load balancer for your cluster. The parameter defaults to `OpenShiftManagedDefault`, which denotes the default internal load balancer. For services defined in an `openshift-kni-infra` namespace, a user-managed load balancer can deploy the `coredns` service to pods in your cluster but ignores `keepalived` and `haproxy` services.
+<2> Required parameter when you specify a user-managed load balancer. Specify the user-managed load balancer's public IP address, so that the Kubernetes API an communicate with the user-managed load balancer.
```

Comment:
```suggestion
<2> Required parameter when you specify a user-managed load balancer. Specify the user-managed load balancer's public IP address, so that the Kubernetes API can communicate with the user-managed load balancer.
```

Reply:
Thanks, Alex. Updated.